ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/Overview.cshtml

```diff
@@ -7,23 +7,19 @@
 	ViewData["Title"] = "Trust Overview";
 	var nonce = HttpContext.GetNonce();
 }
+<style>
+.toggle-content {
```

Comment:
SUGGESTION:

Can you move these styles into a global css file, we will probably need them for the other pagination changes we are going to make

We have ConcernsCaseWork\wwwroot\src\css\_extra.scss

Reply:
Removed as not required